--- serviio-1.4-orig/bin/serviio.sh	2013-12-20 19:41:21.000000000 +0100
+++ serviio-1.4/bin/serviio.sh	2014-01-10 21:57:24.054673860 +0100
@@ -54,7 +54,7 @@
 SERVIIO_CLASS_PATH="$SERVIIO_HOME/lib/*:$SERVIIO_HOME/config"
 
 # Setup Serviio specific properties
-JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Djava.awt.headless=true -Dderby.system.home=$SERVIIO_HOME/library -Dserviio.home=$SERVIIO_HOME"
+JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Djava.awt.headless=true -Dderby.system.home=$SERVIIO_HOME/library -Dserviio.home=$SERVIIO_HOME -Dffmpeg.location=/usr/bin/ffmpeg"
 
 # For Cygwin, switch paths to Windows format before running java
 if $cygwin; then
