--- org/bin/serviio.sh	2014-03-21 22:43:59.000000000 +0100
+++ mod/bin/serviio.sh	2014-03-22 10:06:42.593385730 +0100
@@ -54,7 +54,7 @@
 SERVIIO_CLASS_PATH="$SERVIIO_HOME/lib/*:$SERVIIO_HOME/config"
 
 # Setup Serviio specific properties
-JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Djava.awt.headless=true -Dderby.system.home=$SERVIIO_HOME/library -Dserviio.home=$SERVIIO_HOME"
+JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Djava.awt.headless=true -Dderby.system.home=$SERVIIO_HOME/library -Dserviio.home=$SERVIIO_HOME -Dffmpeg.location=/usr/bin/ffmpeg"
 
 # For Cygwin, switch paths to Windows format before running java
 if $cygwin; then
